SPECS/kernel/config

```diff
@@ -208,7 +208,7 @@ CONFIG_IPC_NS=y
 CONFIG_USER_NS=y
 CONFIG_PID_NS=y
 CONFIG_NET_NS=y
-# CONFIG_CHECKPOINT_RESTORE is not set
+CONFIG_CHECKPOINT_RESTORE=y
```

Comment:
Out of curiosity, do you know if this increases the size of the rpm significantly?

Reply:
RPM size is about 35M before and after the change.
Before: 36276851
After: 36312290

Reply:
Kernel boot time is also essentially unchanged: `Startup finished in 1.364s (kernel)` with the changes and `Startup finished in 1.362s (kernel)` without (from `systemd-analyze`).